db/db.js

```diff
@@ -39,12 +37,25 @@ for (let i = 0; i < sampleData.length; i += 1) {
   });
 }
 
-// console.log(sampleData); 
-
-Comment.find((error, comments) => {
-  if (error) {
-    console.log('could not find comments');
+// Gets all comments for a song on song page
+const getComments = (err, retrievedComments) => {
```

Comment:
Where is the specified song passed in?

Reply:
I didn't generate song data, only the comments that will be on the song page.